src/Markdown.MAML/Transformer/ModelTransformerVersion2.cs

```diff
@@ -201,21 +220,43 @@ private void GatherSyntax(MamlCommand command)
                 {
                     if (parameterSetNames.Count == 1)
                     {
-                        // special case: there is only one parameter set and it's the deafult one
+                        // special case: there is only one parameter set and it's the default one
                         // we don't specify the name in this case.
+                        
                     }
                     else
                     {
+
                         continue;
                     }                    
                 }
+                else if (command.Syntax.Exists(s => s.ParameterSetName == setName))
```

Comment:
revert this change, as explained in the removed comment, we should not use Syntax entry for generating markdown at all